recipe/meta.yaml

```diff
@@ -10,22 +10,22 @@ source:
   sha256: 1ef980abf0e1e072b028d2d76ef95b476632651c96225cf30b619c6eef625672
 
 build:
-  number: 2
+  number: 0
```

Comment:
This should stay 2, right? Otherwise people with conda-forge stream enabled will prefer their build over ours even though ours is newer.

Reply:
I was under the impression it should be reset since this is **_our_** first instance of it.